quinn-proto/src/packet.rs

```diff
@@ -225,6 +297,11 @@ impl Header {
                 w.put_slice(token);
                 w.write::<u16>(0); // Placeholder for payload length; see `set_payload_length`
                 number.encode(w);
+                let pn_pos = 9 + dst_cid.len() + src_cid.len() + token.len();
```

Comment:
Does this handle the variable-length encoding of `token.len()`?

Reply:
I had actually already fixed that, but hadn't pushed it yet.